scripts/rhv_tier_analyzer.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+import sys
+
+PROVIDER = 'rhv_cfme_integration'
+TEST_PATH = '../cfme/tests'
+TIER_MARKER = '-m rhv1 or rhv2 or rhv3'
+
+
+def check_virtualenv():
+    """Check if we are in virtualenv and if not, exit."""
+    if not hasattr(sys, 'real_prefix'):
+        print('You must activate CFME virtualenv in oder to run this script.')
+        sys.exit(1)
+
+
+def get_pytest_output(use_tier_marker=False):
+    """Get string output of pytest command."""
+    print('Getting pytest output {} tier marker.'.format('with' if use_tier_marker else 'without'))
+    args = ['pytest', TEST_PATH, '--collect-only', '--long-running', '--use-provider', PROVIDER]
+    if use_tier_marker:
+        args.append(TIER_MARKER)
+
+    return subprocess.check_output(args)
+
+
+def parse_pytest_output(output):
+    """Parse raw pytest output to get list of parametrized test cases."""
+    list_of_lines = output.splitlines()
+    list_of_functions = [l.strip() for l in list_of_lines if l.strip().startswith('<Function ')]
+    integration_only = [l for l in list_of_functions if PROVIDER in l]
+
+    test_cases = []
+    for func in integration_only:
```

Comment:
**Optional**

Consider using a regex expression here with a named matching group to pull out the test case string here.